api/resource/iam_policy.rb

```diff
@@ -39,10 +39,28 @@ class IamPolicy < Api::Object
       # others expect POST requests
       attr_reader :fetch_iam_policy_verb
 
+      # Last part of URL for fetching IAM policy.
+      attr_reader :fetch_iam_policy_method
+
+      # Some resources allow retrieving the IAM policy with GET requests,
```

Comment:
Swap `/retrieving/setting/` and `/GET/POST/`

Reply:
Whoops, done